### PR TITLE
228 plugins refactoring

### PR DESCRIPTION
**Pull request by roma-glushko**
The general refactoring of plugin classnames has been performed.

### Description
1. Moved all our plugins to Plugins namespace
2. Leaded our namespace to ModuleName\Plugins\{CustomizedModule}
3. Provided more clear names of our plugins (describe behaviour in class name)

### Fixed Issues (if relevant)
1. magento-engcom/msi#228: Global refactoring of plugin namespaces and names

### CODE REVIEW

File: app/code/Magento/InventoryCatalog/Plugins/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php [hunk 1]
```diff
@@ -4,21 +4,20 @@
  * See COPYING.txt for license details.
  */
 
-namespace Magento\InventoryCatalog\Plugin\Model;
+namespace Magento\InventoryCatalog\Plugins\Inventory\Reservation;
```
Comment (naydav) [inline app/code/Magento/InventoryCatalog/Plugins/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php]: Generally in Magento plugins directory should be `Plugin` (single form but not plurar)

**Review by naydav — CHANGES_REQUESTED** [review]

File: app/code/Magento/InventoryCatalog/Plugins/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php (hunk 1 above)
Comment (roma-glushko) [reply app/code/Magento/InventoryCatalog/Plugins/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php]: This refactoring has been performed. Thank you!

**Comment by roma-glushko** [comment]
Hey, Travis has mentioned a couple of other issues. It doesn't look like they are related to changes  somehow. Anyway, please let me know if they should be fixed in scope of this task.
Thanks.

**Comment by naydav** [comment]
@roma-glushko 

Hello, sorry for delay in answer
Firstly need to merge develop branch, we have a lot of changes

After that, we could continue work on Travis
Thanks

File: app/code/Magento/InventoryCatalog/Plugin/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php [hunk 2]
```diff
@@ -30,11 +30,13 @@ class UpdateLegacyCatalogInventoryPlugin
     private $productRepository;
 
     /**
-     * @var \Magento\CatalogInventory\Api\StockRegistryInterface
+     * @var StockRegistryInterface
      */
     private $stockRegistry;
 
     /**
+     * UpdateLegacyCatalogInventoryPlugin constructor.
```
Comment (naydav) [inline app/code/Magento/InventoryCatalog/Plugin/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php:38]: Pls remove this extra comment
It is clear that is constructor

File: app/code/Magento/InventoryCatalog/Plugin/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php [hunk 3]
```diff
@@ -3,22 +3,22 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
 
-namespace Magento\InventoryCatalog\Plugin\Model;
+namespace Magento\InventoryCatalog\Plugin\Inventory\Reservation;
```
Comment (naydav) [inline app/code/Magento/InventoryCatalog/Plugin/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php:8]: Looks like need to move to `InventoryApi` namespace

File: app/code/Magento/InventoryCatalog/Plugin/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php [hunk 4]
```diff
@@ -56,20 +58,20 @@ public function __construct(
...
     {
         $this->updateStockItemAndStatusTable($reservations);
-        return $result;
     }
 
     /**
      * Updates cataloginventory_stock_item and cataloginventory_stock_status qty with reservation information.
      *
      * @param ReservationInterface[] $reservations
+     *
```
Comment (naydav) [inline app/code/Magento/InventoryCatalog/Plugin/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php:74]: Generally, we don't provide extra line between `param` and `return` annotations

File: app/code/Magento/InventoryCatalog/Plugin/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php [hunk 5]
```diff
@@ -56,20 +58,20 @@ public function __construct(
...
      * @return void
      */
     private function updateStockItemAndStatusTable(array $reservations)
```
Comment (naydav) [inline app/code/Magento/InventoryCatalog/Plugin/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php:77]: Looks like we can move this code in `afterExecute`
We don't have advantage in separate private method in this case

File: app/code/Magento/InventoryCatalog/Test/Integration/UpdateLegacyCatalogInventoryPluginTest.php [hunk 6]
```diff
@@ -21,6 +22,9 @@
 use Magento\Indexer\Model\Indexer;
 use Magento\Inventory\Indexer\SourceItem\SourceItemIndexer;
 
+/**
+ * Class UpdateLegacyCatalogInventoryPluginTest
```
Comment (naydav) [inline app/code/Magento/InventoryCatalog/Test/Integration/UpdateLegacyCatalogInventoryPluginTest.php:26]: It is more useful to provide doctype about functionality but not about class name

File: app/code/Magento/InventorySales/Plugin/Inventory/Stock/EnrichStockItemDataBySalesChannelsDataPlugin.php [hunk 7]
```diff
@@ -14,9 +14,9 @@
 use Psr\Log\LoggerInterface;
 
 /**
- * Load Sales Channels for Stock on Get method of StockRepositoryInterface
+ *  Enriches Stock Data by Sale Channels Data on StockRepositoryInterface::get()
```
Comment (naydav) [inline app/code/Magento/InventorySales/Plugin/Inventory/Stock/EnrichStockItemDataBySalesChannelsDataPlugin.php:17]: Generally, in Magento, we use `load` word for such cases

**Review by naydav — CHANGES_REQUESTED** [review]

**Comment by naydav** [comment]
Closed due to changes in mainline (We have finished refactoring in scope of Kyiv Contribution Day)

**Pull request closed without merging**